styles.css

```diff
@@ -63,6 +66,10 @@ main {
   text-decoration: none;
 }
 
```

Comment:
- [ ] Also you may consider readjusting the look of your button on mobile version

Your design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/201392949-5581ca15-2d7a-4248-93cf-733b82a760e9.png)

Figma template :point_down: 
![image](https://user-images.githubusercontent.com/97100091/201393051-5e394d8f-e89b-456e-be2c-56a1cc61da8b.png)